README.md

```diff
@@ -1,6 +1,22 @@
 # aider-environment
 
-run
+## note about docker
+
+If you are not familliar with docker, is better before running script from this repository ensure, that you have configured and running docker. Try run command:
```

Comment:
@srbt how about:

## Note about Docker

If you are unfamiliar with Docker, it's important to ensure that Docker is configured and running on your system before you run any scripts from this repository. You can verify this by running the following command: